app/code/core/Mage/Paypal/Model/Payflow/Request.php

```diff
@@ -45,15 +45,13 @@ public function __call($method, $args)
         switch (substr($method, 0, 3)) {
             case 'get' :
                 //Varien_Profiler::start('GETTER: '.get_class($this).'::'.$method);
-                $data = $this->getData($key, isset($args[0]) ? $args[0] : null);
                 //Varien_Profiler::stop('GETTER: '.get_class($this).'::'.$method);
-                return $data;
+                return $this->getData($key, isset($args[0]) ? $args[0] : null);
 
             case 'set' :
                 //Varien_Profiler::start('SETTER: '.get_class($this).'::'.$method);
-                $result = $this->setData($key, isset($args[0]) ? $args[0] : null);
                 //Varien_Profiler::stop('SETTER: '.get_class($this).'::'.$method);
-                return $result;
+                return $this->setData($key, isset($args[0]) ? $args[0] : null);
```

Comment:
In these cases it looks like this was done for the purpose of being able to uncomment the Varien_Profiler. IMHO these lines should either be uncommented and the variable assignment kept as-is or the lines should be removed.

Reply:
Removed these lines.